content/platforms/kubernetes/db-controller.md

```diff
@@ -0,0 +1,274 @@
+
+```sh
+kubectl port-forward guestbook-667fcbf6f6-gztjv 8080:80
+```
+
+Browse to `http://localhost:8080/` to view the demonstration.
+
+
+## Options for Databases
+
+The database options are specified in the `spec` section of the database custom resource.
+These options include options that you can change and options that are created by the controller for applications or developers. For example, the database connection information is stored in a secret.
+
+### `databaseSecretName`
+
+A string containing the name of a secret that contains the desired database password.
+
+The default value is the database name (`metadata.name`) with "redb-" as a prefix.
+
+To create your own database password in the secret, run:
+
+```sh
+ kubectl create secret generic mydb --from-literal=password=<password>
+ ```
+
+where <password> is your desired password in plain text.
+
+In the database custom resource, the value is just secret name. In this example,
+the name is `mydb`:
+
+ ```yaml
+ databaseSecretName: mydb
+ ```
+
+When the database is created, the secret is updated to include the port and service name for the database
+but the password will remain the same.
+
+### `enforceClientAuthentication`
+
+A boolean that indicates whether client authentication should be enforced (default: `true`).
+
+### `evictionPolicy`
+
+An [eviction policy](https://docs.redislabs.com/latest/rs/administering/database-operations/eviction-policy/) (default: `volitile-lru`)
+
+### `memorySize`
+
+The amount of memory to allocate that is at least 100MB. Values are an integer
+suffixed with a unit. For example, values like 1GB, 250MB, etc.
+
+### `persistence`
+
+The value for the [database persistence](https://docs.redislabs.com/latest/rs/concepts/data-access/persistence/) setting.
+
+The value is a keyword with the values:
+
+| Value | Description |
+| ----- | ----------- |
+| disabled | Data is not persisted (default) |
+| AofEverySecond | Data is synced to disk every second |
+| AofAlways | Data is synced to disk with every write. |
+| SnapshotEveryHour | A snapshot of the database is created every hour |
+| SnapshotEvery6Hour | A snapshot of the database is created every 6 hours. |
+| SnapshotEvery12Hour | A snapshot of the database is created every 12 hours. |
+
+### `rackAware`
+
+A boolean that indicates whether the database is [rack-zone aware](https://docs.redislabs.com/latest/rs/concepts/high-availability/rack-zone-awareness/) (default: the cluster setting)
+
+### `redisEnterpriseCluster`
+
+The name of the cluster to create the database on.
+
+The value has a single `name` property.
+For example, to refer to the `rec` cluster:
+
+```YAML
+redisEnterpriseCluster:
+   name: rec
+```
+
+### `replication`
+
+A boolean that indicates whether in-memory database replication is enabled (default: `false`).
+
+When enabled, the database has a replica shard for every master.
+
+### `sharedCount`
+
+The number of database shards (default: `1`).
```

Comment:
Please add reference to a relevant RS article

Reply:
added link